dynamic_rest/serializers.py

```diff
@@ -510,7 +510,7 @@ class WithDynamicModelSerializerMixin(WithDynamicSerializerMixin):
 
     @classmethod
     def get_model(cls):
-        return cls.Meta.model
+        return getattr(cls.Meta, 'model', None)
```

Comment:
This would still raise an error if the serializer has no Meta class, since we're calling `cls.Meta`.